.github/workflows/update_gas_prices.yml

```diff
@@ -35,12 +35,7 @@ jobs:
         run: python update_gas_prices.py
 
       - name: 📎 Commit gas information with history
-        run: |
-          git config --global user.name 'Dntfreitas'
-          git config --global user.email 'dntfreitas123@gmail.com'
-          git fetch && git pull --ff-only
-          git add gas_info.json
-          git add history/gas_info_history.csv
-          git add history/gas_info_history.json
-          git diff-index --quiet HEAD || git commit -m "(⛽️): updated on `date +'%Y-%m-%d %H:%M:%S'`"
-          git push
+        uses: actions-js/push@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: main
```

Comment:
We can also add the option:
```yml
message: "(⛽️): updated on `date +'%Y-%m-%d %H:%M:%S'`"
```

Reply:
What do you think @Dntfreitas ?

Reply:
I tried, by it doesn't seem to work :( 
The default message, however, contais the timestamp info

Reply:
Ok then we can move on I guess.